torch/utils/model_dump/code.js

```diff
@@ -54,21 +54,14 @@ class Hider extends Component {
     this.setState({ shown: this.props.shown === "true" });
   }
 
-  render(_, {shown}) {
-    return html`<h2>
-      <span class=caret onClick=${() => this.click()} >${caret(shown)} </span>
-      ${this.props.name}</h2>`;
+  render({name, children}, {shown}) {
+    let my_caret = html`<span class=caret onClick=${() => this.click()} >${caret(shown)}</span>`;
+    return html`<h2>${my_caret} ${name}</h2>
+      ${shown ? this.props.children : []}`;
```

Comment:
ah! interesting. So you basically just set the html to whatever `this.props.children` has or an empty array.